Dockerfile.android

```diff
@@ -4,18 +4,18 @@ FROM godot-mono:${img_version}
 ARG mono_version
 
 ENV ANDROID_SDK_ROOT=/root/sdk
-ENV ANDROID_NDK_VERSION=21.3.6528147
+ENV ANDROID_NDK_VERSION=21.4.7075529
 ENV ANDROID_NDK_ROOT=${ANDROID_SDK_ROOT}/ndk/${ANDROID_NDK_VERSION}
 
 RUN if [ -z "${mono_version}" ]; then echo -e "\n\nargument mono-version is mandatory!\n\n"; exit 1; fi && \
     dnf -y install --setopt=install_weak_deps=False \
       java-1.8.0-openjdk-devel ncurses-compat-libs && \
-    mkdir -p sdk/cmdline-tools && cd sdk/cmdline-tools && \
-    curl -LO https://dl.google.com/android/repository/commandlinetools-linux-6609375_latest.zip && \
-    unzip commandlinetools-linux-6609375_latest.zip && \
-    rm commandlinetools-linux-6609375_latest.zip && \
-    yes | tools/bin/sdkmanager --licenses && \
-    tools/bin/sdkmanager --sdk_root="${ANDROID_SDK_ROOT}" "ndk;${ANDROID_NDK_VERSION}" 'cmdline-tools;latest' 'build-tools;30.0.1' 'platforms;android-30' 'cmake;3.10.2.4988404'
+    mkdir -p sdk && cd sdk && \
+    curl -LO https://dl.google.com/android/repository/commandlinetools-linux-6858069_latest.zip && \
+    unzip commandlinetools-linux-6858069_latest.zip && \
+    rm commandlinetools-linux-6858069_latest.zip && \
+    yes | cmdline-tools/bin/sdkmanager --sdk_root="${ANDROID_SDK_ROOT}" --licenses && \
```

Comment:
@m4gr3d FYI we need to update setup instructions as the current cmdline tools unzip as `cmdline-tools/bin/sdkmanager`, and it's not a valid SDK setup (requires `latest` folder) so to accept licenses one also needs to pass `--sdk_root`.

Reply:
@akien-mga Good catch! I've updated https://github.com/godotengine/godot-docs/pull/4522 and https://github.com/godotengine/godot-docs/pull/4673 with the info.